homeassistant/components/broadlink/switch.py

```diff
@@ -91,11 +92,12 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
 
     devices = config.get(CONF_SWITCHES)
     slots = config.get("slots", {})
-    ip_addr = config.get(CONF_HOST)
+    host = config.get(CONF_HOST)
+    mac_addr = config.get(CONF_MAC)
     friendly_name = config.get(CONF_FRIENDLY_NAME)
-    mac_addr = binascii.unhexlify(config.get(CONF_MAC).encode().replace(b":", b""))
     switch_type = config.get(CONF_TYPE)
     retry_times = config.get(CONF_RETRY)
+    dev_type = 0x5F36 if switch_type in ["rm_mini3_5f36", "rm4"] else 0x272A
```

Comment:
I would recommend to create constants for the list and each device type to make it easier to read, because right now I have no idea what device type `0x5F36` is for example.

Reply:
We don't allow this level of protocol information in home assistant. We can't add this.